changelog/2.0.0_2022-11-30/mount-ids.md

```diff
@@ -0,0 +1,34 @@
+Bugfix: Make storage users mount ids unique by default
+
+The mount ID of the storage users provider needs to be unique by default. We made this value configurable and added it to ocis init to be sure that we have a random uuid v4. This is important for federated instances.
+
+> **Warning**
+>BREAKING Change: In order to  make every ocis storage provider ID unique by default, we needed to use a random uuidv4 during ocis init. Existing installations need to set this value explicitly or ocis will terminate after the upgrade.
+> To upgrade from 2.0.0-rc.1 to 2.0.0-rc.2 you need to set `GATEWAY_STORAGE_USERS_MOUNT_ID` and `STORAGE_USERS_MOUNT_ID` to the same random uuidv4.
```

Comment:
```suggestion
> To upgrade from 2.0.0-rc.1 to 2.0.0-rc.2, 2.0.0 or later you need to set `GATEWAY_STORAGE_USERS_MOUNT_ID` and `STORAGE_USERS_MOUNT_ID` to the same random uuidv4.
```

Reply:
Yeah, had the same impression, we should not give the impression of needing to upgrade from rc.1 to rc.2 first and then to 2.0.0 final.

Reply:
thank you. Will add this.